deployments/k8s/README.md

```diff
@@ -22,6 +22,13 @@ A few things to do before deploying these directly (when **not using Helm**):
 
 	`kubectl create secret generic --from-literal access-token=MY_ACCESS_TOKEN signalfx-agent`
 
+ 4. Doublecheck the `daemonset.yaml` included stanza and remove if not consistent with your environment.
```

Comment:
do you have to just remove it if it's not the same as your environment, or make sure that it's consistent with your environment?

Reply:
@slernersplunk what other values were you seeing for this?  The problem is if you remove it and you have Windows nodes, it will make a bunch of errors trying to deploy the Linux agent to Windows.

Reply:
Another RedHat discovery- our default entry for node selector tanked the SmartAgent pods... removing it fixed the problem...  I'm not saying we SHOULD remove it- i'm saying IF that nodeselector is inconsistent with one's environment, remove it...

Reply:
Trying to get these things that block us for 3 hours but are only a single config change into our docs as suggestions so people can find them- btw this was also in OpenShift